apps/emqx_bridge/src/emqx_bridge_api.erl

```diff
@@ -1079,7 +1079,9 @@ call_operation(NodeOrAll, OperFunc, Args = [_Nodes, BridgeType, BridgeName]) ->
         {error, {unhealthy_target, Message}} ->
             ?BAD_REQUEST(Message);
         {error, Reason} when not is_tuple(Reason); element(1, Reason) =/= 'exit' ->
-            ?BAD_REQUEST(redact(Reason))
+            ?BAD_REQUEST(redact(Reason));
+        {error, Reason} ->
+            ?BAD_REQUEST(Reason)
```

Comment:
keep only one clause ?

Reply:
Fixed in: [d14d0fb](https://github.com/emqx/emqx/pull/11869/commits/d14d0fbcb1ec856d283062bc0decf6add02c776b)